products/netbsd.md

```diff
@@ -0,0 +1,73 @@
+---
+permalink: /netbsd
+title: NetBSD
+category: os
+versionCommand: uname -r
+activeSupportColumn: false
+releasePolicyLink: https://www.netbsd.org/releases/
+releaseDateColumn: true
+releaseColumn: false
+releases:
+-   releaseCycle: "9"
+    releaseDate: 2022-02-14
+    latest: "9.3"
+    latestRelaseDate: 2022-08-04
+    link: https://www.netbsd.org/releases/formal-9/NetBSD-9.3.html
```

Comment:
these links can be removed after adding a suitable changelogTemplate
[if helps needed i can help you about this one]

Reply:
@usta Thanks for your time and good suggestion! I gave it a try with the latest commit on the [NetBSD branch](https://github.com/alexkoster/endoflife.date/tree/netbsd) but it appears that only non-EOL releases now have their changelog linked. Should this be the expected behavior?  See the screenshot below:

![Screenshot 2022-10-30 at 01-13-28 NetBSD](https://user-images.githubusercontent.com/8161603/198868995-e5e5fdf2-0e38-4e60-8aff-160797e08d2f.png)

Reply:
We are not showing EOL'ed version's links but its link tags already exist its json api output

Reply:
one small reminder you dont need to mention the branch/and its link in each of your comments :))

Reply:
> We are not showing EOL'ed version's links but its link tags already exist its json api output

Thanks for clarifying!



> one small reminder you dont need to mention the branch/and its link in each of your comments :))

And I appreciate you saving me some keystrokes on this keyboard that needs replacing (then again, many of those typos and bugs are sourced from the PEBKAC)